plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/SpanAddingClassVisitorFactory.kt

```diff
@@ -99,6 +100,9 @@ abstract class SpanAddingClassVisitorFactory :
                         sentryModulesService.isNewDatabaseInstrEnabled() ||
                             sentryModulesService.isOldDatabaseInstrEnabled()
                     },
+                    OkHttpEventListener().takeIf {
+                        sentryModulesService.isOkHttpInstrEnabled()
```

Comment:
a little bit late to the party, but I think this one is not what we need. We added the new `SentryOkHttpEventListener` in version 6.20.0, whereas this one checks for version 5.0.0. so we'd have to introduce new `isOkHttpListenerInstrEnabled` or something like that

Reply:
you're right 🤔 
how I missed it...

Reply:
https://github.com/getsentry/sentry-android-gradle-plugin/pull/508